libcudacxx/include/cuda/__execution/require.h

```diff
@@ -52,12 +52,17 @@ struct __get_requirements_t
 _CCCL_GLOBAL_CONSTANT auto __get_requirements = __get_requirements_t{};
 
 template <class... _Requirements>
-[[nodiscard]] _CCCL_TRIVIAL_API auto require(_Requirements... __requirements)
+[[nodiscard]] _CCCL_TRIVIAL_API auto require(_Requirements...)
 {
   static_assert((_CUDA_VSTD::is_base_of_v<__requirement, _Requirements> && ...),
                 "Only requirements can be passed to require");
 
-  return _CUDA_STD_EXEC::prop{__get_requirements_t{}, _CUDA_STD_EXEC::env{__requirements...}};
+  // clang < 19 doesn't like this code
+  // since the only requirements we currently allow are in determinism.h and
+  // all of them are stateless, let's ignore incoming parameters
+  _CUDA_STD_EXEC::env<_Requirements...> __env{};
```

Comment:
Important: Let's help our future selves by asserting this property then in case we ever make requirements stateful:
```c++
static_assert((::cuda::std::is_empty_v<_Requirements> && ...), "Stateful requirements are not implemented");
```